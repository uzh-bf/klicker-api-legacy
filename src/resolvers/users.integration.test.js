const request = require('supertest')

const server = require('../app')

describe('users', () => {
  it('works', async () => {
    const response = await request(server).post('/graphql', {})
    expect(response).toMatchSnapshot()
  })
})
