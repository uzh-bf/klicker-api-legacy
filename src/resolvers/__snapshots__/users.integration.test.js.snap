// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`users works 1`] = `
Object {
  "header": Object {
    "access-control-allow-credentials": "true",
    "access-control-allow-origin": "http://localhost:3000",
    "connection": "close",
    "content-length": "748",
    "content-security-policy": "default-src 'self'",
    "content-type": "text/html; charset=utf-8",
    "date": "Tue, 21 Nov 2017 09:17:05 GMT",
    "vary": "Origin, Accept-Encoding",
    "x-content-type-options": "nosniff",
    "x-dns-prefetch-control": "off",
    "x-download-options": "noopen",
    "x-frame-options": "SAMEORIGIN",
    "x-xss-protection": "1; mode=block",
  },
  "req": Object {
    "data": undefined,
    "headers": Object {
      "user-agent": "node-superagent/3.8.1",
    },
    "method": "POST",
    "url": "http://127.0.0.1:55290/graphql",
  },
  "status": 500,
  "text": "<!DOCTYPE html>
<html lang=\\"en\\">
<head>
<meta charset=\\"utf-8\\">
<title>Error</title>
</head>
<body>
<pre>Error: Must provide document<br> &nbsp; &nbsp;at invariant (c:\\\\Users\\\\Roland\\\\git\\\\ibf\\\\klicker-api\\\\node_modules\\\\graphql\\\\jsutils\\\\invariant.js:18:11)<br> &nbsp; &nbsp;at Object.validate (c:\\\\Users\\\\Roland\\\\git\\\\ibf\\\\klicker-api\\\\node_modules\\\\graphql\\\\validation\\\\validate.js:58:34)<br> &nbsp; &nbsp;at doRunQuery (c:\\\\Users\\\\Roland\\\\git\\\\ibf\\\\klicker-api\\\\node_modules\\\\apollo-server-core\\\\dist\\\\runQuery.js:88:38)<br> &nbsp; &nbsp;at c:\\\\Users\\\\Roland\\\\git\\\\ibf\\\\klicker-api\\\\node_modules\\\\apollo-server-core\\\\dist\\\\runQuery.js:22:54<br> &nbsp; &nbsp;at &lt;anonymous&gt;<br> &nbsp; &nbsp;at process._tickCallback (internal/process/next_tick.js:188:7)</pre>
</body>
</html>
",
}
`;
